app/presenters/openstack_presenter.rb

```diff
@@ -0,0 +1,14 @@
+class OpenstackPresenter < ComputeResourcePresenter
```

Comment:
I would recommend either nesting it inside `ComputeResourcePresenter::Openstack` or prefixing it with `ComputeResourceOpenstackPresenter`, so that it's clear what it belongs to: it might get messy once we use this pattern more in the future

Reply:
I think we should use a path that can be autoloaded `app/presenters/compute_resource_presenter/openstack.rb` -> `ComputeResourcePresenter ::OpenStack`.